.circleci/RunPyTests.sh

```diff
@@ -39,7 +39,7 @@ run_pytest_cmd () {
     cwd_mount="-v ${PWD}/tests:/rundir:rw"
     PYTEST_RUN="docker run --rm -ti -u $(id -u) -w /rundir --entrypoint pytest "
     PYTEST_RUN+="${workdir_mount} ${patch_mount} ${cwd_mount} ${cfg_arg} ${bids_mount} ${output_mount} ${IMAGE} "
-    PYTEST_RUN+="--data_dir=/bids-input --output_dir=/out participant --working_dir=/work /rundir"
+    PYTEST_RUN+="--data_dir=/bids-input --output_dir=/out --working_dir=/work /rundir/"
```

Comment:
Having `participant` here led to an error when I ran `RunPyTests.sh` locally. Does it work for you @kahinimehta?